File: TugasPraktikum03/Bahasa Python/PRAK302-2310817320013-AikoAnatashaWendiono.py
huruf = int(input("Input:\n"))

if (huruf >= 80):
    print("Output:\nA")
elif (70 <= huruf <= 79):
    print("Output:\nB")
elif (60 <= huruf <= 69):
    print("Output:\nC")
elif (50 <= huruf <= 59):
    print("Output:\nD")
elif (huruf < 50):
    print("Output:\nE")

File: TugasPraktikum03/Bahasa C/PRAK302-2310817320013-AikoAnatashaWendiono.c
#include <stdio.h>

int main() {
    int huruf;

    printf("Input:\n");
    scanf("%d", &huruf);

    if (huruf >= 80) {
        printf("Output:\nA\n");
    } else if (huruf >= 70) {
        printf("Output:\nB\n");
    } else if (huruf >= 60) {
        printf("Output:\nC\n");
    } else if (huruf >= 50) {
        printf("Output:\nD\n");
    } else {
        printf("Output:\nE\n");
    }

    return 0;
}
